ocfweb/docs/docs/staff/scripts/sorry.md

```diff
@@ -0,0 +1,28 @@
+[[!meta title="sorry: ban/disable an OCF account"]]
+
+## Introduction
+
+`sorry` is the command used to ban or disable OCF accounts. Accounts are banned 
+for various reasons (users/groups being disruptive or otherwise violating 
+OCF/university policies, security issues, lack of contact information, etc.) 
+and alumni sometimes request to have their accounts disabled to stop vhosts, 
+take down information, or for myriad other reasons.
+
+## Usage/Example
+
+    Usage: sorry [user to be sorried] [sorry file]
```

Comment:
No, "sorry file" refers to a text file containing the reason the account was sorried. It is included in an email to the sorried user. The sorry files are located in `/opt/share/utils/staff/acct/sorry/`.